custom_components/meural/config_flow.py

```diff
@@ -22,13 +20,7 @@ async def validate_input(hass: core.HomeAssistant, data):
     Data has the keys from DATA_SCHEMA with values provided by the user.
     """
     session = hass.helpers.aiohttp_client.async_get_clientsession()
-    try:
-        token = await pymeural.authenticate(session, data["email"], data["password"])
-    except (aiohttp.ClientError, asyncio.TimeoutError):
```

Comment:
These exceptions are now no longer handled or translated to the config flow specific exceptions CannotConnect, InvalidAuth.

Reply:
nevermind, see now you import them.